### PR TITLE
Update country codes

### PR DESCRIPTION
**Pull request by jjeffryes**
This adds the United States Minor Outlying Islands, and updates the names of Eswatini and Việt Nam.

It should be merged in conjunction with the matching server update.

https://github.com/OpenBazaar/openbazaar-go/pull/1175

### CODE REVIEW

**Comment by jjeffryes** [comment]
The server code related to this PR has been merged.

**Comment by rmisio** [comment]
> The server code related to this PR has been merged.

I'll start working through the PR reviews later today.

File: js/languages/en_US.json [hunk 1]
```diff
@@ -2345,11 +2345,12 @@
     "UNITED_ARAB_EMIRATES": "United Arab Emirates (The)",
     "UNITED_KINGDOM": "United Kingdom of Great Britain and Northern Ireland (The)",
     "UNITED_STATES": "United States of America (The)",
+    "UNITED_STATES_MINOR_ISLANDS": "United States Minor Outlying Islands",
     "URUGUAY": "Uruguay",
     "UZBEKISTAN": "Uzbekistan",
     "VANUATU": "Vanuatu",
     "VENEZUELA": "Venezuela (Bolivarian Republic Of)",
-    "VIETNAM": "Vietnam",
+    "VIETNAM": "Việt Nam",
```
Comment (rmisio) [inline js/languages/en_US.json]: Should this really be localized like this? The other countries are in English and it is up to the translation files to localize.

Comment (jjeffryes) [reply js/languages/en_US.json]: I saw it that way on the ISO site I was checking the data against. But checking elsewhere it should be left as the English version. I'll update it.

**Pull request merged**